package.json

```diff
@@ -23,35 +23,34 @@
   "bugs": {
     "url": "https://github.com/fatso83/grunt-penthouse/issues"
   },
+  "license": "MIT",
   "licenses": [
     {
       "type": "MIT",
       "url": "https://github.com/fatso83/grunt-penthouse/blob/master/LICENSE-MIT"
     }
   ],
   "engines": {
-    "node": ">= 0.8.0"
-  },
-  "scripts": {
-    "test": "grunt test"
+    "node": ">= 0.12.0"
   },
   "dependencies": {
-    "penthouse": ">= 0.5.0 <1.0.0)"
```

Comment:
Not too sure about this. You are pinning the grunt plugin to a specific release, while it would work with any future as it was. Please elaborate on why this is a good idea. I had way too many PRs in the past asking me to update the grunt plugin just to update the version number.